nucleus/core/kernel/pom.xml

```diff
@@ -258,7 +258,11 @@
             <groupId>org.glassfish</groupId>
             <artifactId>javax.el</artifactId>
         </dependency>
-        
+        <dependency>
```

Comment:
Does this change have potential to impact test suites? - I recall a problem in testing in the past ~3 weeks to do with tiger-types

Reply:
This what has been fixed in 2.6.1 of HK2 - it has an OSGi import of tiger-types that was unused within it and has now been removed. See eclipse-ee4j/glassfish-hk2#469.